src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,64 @@
+package duke;
```

Comment:
I agree! You have clearly abstracted out the different classes with different functions.